config/deploy.rb

```diff
@@ -0,0 +1,55 @@
+# config valid only for current version of Capistrano
+# lock '3.4.0'
+
+set :application, 'figgy'
+set :repo_url, 'https://github.com/pulibrary/figgy.git'
+
+# Default branch is :master
+# ask :branch, `git rev-parse --abbrev-ref HEAD`.chomp
+set :branch, ENV['BRANCH'] || 'master'
+
+# Default deploy_to directory is /var/www/my_app_name
+# set :deploy_to, '/var/www/my_app_name'
+set :deploy_to, '/opt/figgy'
```

Comment:
I like /opt/figgy personally. We only had rails_app because the puppet recipe was easier to write that way, but I think we have that solved in ansible. Maybe we can ask everyone's opinion?